frontend/libs/studio-components/src/components/StudioSearch/StudioSearch.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef } from 'react';
+import { Label, Search, type SearchProps } from '@digdir/designsystemet-react';
+import type { WithoutAsChild } from '../../types/WithoutAsChild';
+import { StudioFieldset } from '../StudioFieldset';
+
+export type StudioSearchProps = WithoutAsChild<SearchProps> & { legend?: string };
+
+const StudioSearch = forwardRef<HTMLInputElement, StudioSearchProps>(
+  ({ size = 'sm', label, legend, ...rest }, ref) => {
+    const showLabel = !!label;
+    return showLabel ? (
+      <StudioFieldset legend={legend}>
+        <Label htmlFor='searchId'>{label}</Label>
+        <Search {...rest} id='searchId' size={size} ref={ref} />
```

Comment:
Vi bør ikke hardkode ID-en. En ID må være unik, så dette vil gjøre koden ugyldig hvis siden inneholder flere søkefelter. Derfor bør vi generere en unik ID med `useId`. Vi bør også sjekke om det allerede er satt in ID i proppene. I så fall er det den som skal gjelde.